host/src/FunctionsNetHost/Grpc/IncomingGrpcMessageHandler.cs

```diff
@@ -87,7 +87,13 @@ private async Task Process(StreamingMessage msg)
 
                     Logger.LogTrace($"Will wait for worker loaded signal.");
                     WorkerLoadStatusSignalManager.Instance.Signal.WaitOne();
-                    Logger.LogTrace($"Received worker loaded signal. Forwarding environment reload request to worker.");
+
+                    var logMessage = $"FunctionApp assembly loaded successfully. ProcessId:{Environment.ProcessId}";
+                    if (OperatingSystem.IsWindows())
+                    {
+                        logMessage += $", AppPoolId:{Environment.GetEnvironmentVariable(EnvironmentVariables.AppPoolId)}";
+                    }
+                    Logger.Log(logMessage);
```

Comment:
nit: everywhere else in this file uses Logger.LogTrace.
not sure if there is any difference or not.

Reply:
Entries with Logger.Trace will be logged only when `AZURE_FUNCTIONS_FUNCTIONSNETHOST_TRACE` env variable is set to `1`. Logger.Log will be alwys logged (more like LogInfo). There is some planned improvement to this logging code and we can fix the name to be more readable at that point.